.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -56,6 +56,42 @@ jobs:
       - name: shellcheck
         run: shellcheck scripts/*.sh
 
+  coverage:
+    # Run after pre checks
+    needs: [ clang_format, license_check, rust_format_check, markdown_lint, shellcheck ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Setup toolchain
+        run: |
+          cargo install grcov
+          rustup component add llvm-tools
+      - name: build
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
+          LLVM_PROFILE_FILE: coverage/coverage_data-%p-%m.profraw
+        run: |
+          cd crates/cxx-qt-gen
+          cargo build
```

Comment:
now that you have the `ignore` setup, can you try using `cargo build --package cxx-qt-gen` again so this is one line?